ProjectHub2/services/mongodbService.js

```diff
@@ -0,0 +1,51 @@
+var MongoClient = require('mongodb').MongoClient;
+
+var url = 'mongodb://localhost:27017/data';
+
+var mongoWrapper = (usedCollection, callback) => {
+    MongoClient.connect(url, function (err, db) {
+        console.log("Connected correctly to server.");
+        var collection = db.collection(usedCollection);
+        callback(collection);
+        db.close();
+    });
+};
+
+var insertItems = (itemsArray, usedCollection) => {
+    mongoWrapper(usedCollection, (collection) => {
```

Comment:
maybe I'm not used to such way, but I find a bit cofusing. I'd done something like
```javascript
var insertItems = (itemsArray, usedCollection) => {
    var saveItemsCallback = (collection) => { 
        itemsArray.forEach(function (item) {
            collection.save(item);
        });
    };
    mongoWrapper(usedCollection, saveItemsCallback);
});
```
On the other hand, I agree someone can find confusing my way too